src/test/java/org/broadinstitute/hellbender/tools/spark/pipelines/SortSamSparkIntegrationTest.java

```diff
@@ -61,13 +59,32 @@ public void test() throws Exception {
         final File sortedBam = new File(getTestDataDir(), "count_reads_sorted.bam");
```

Comment:
This test method needs a better name than just `test()`. Also, why do we need this test case in addition to `testSortBAMs()`? Can they be unified?